### PR TITLE
Add config file support for cli

### PR DESCRIPTION
**Pull request by antekresic**

### CODE REVIEW

File: pkg/runner/runner.go [hunk 1]
```diff
@@ -80,7 +82,10 @@ func ParseFlags(cfg *Config, args []string) (*Config, error) {
 	envy.Parse("PROMSCALE")
 
 	// Ignore errors; CommandLine is set for ExitOnError.
-	_ = flag.CommandLine.Parse(args)
+	_ = ff.Parse(flag.CommandLine, args,
+		ff.WithConfigFile("config.yml"),
```
Comment (Harkishen-Singh) [inline pkg/runner/runner.go:86]: Shouldn't this name be supplied from CLI?

Comment (cevian) [reply pkg/runner/runner.go:86]: Yeah I think using `WithConfigFileFlag` is better and define a new flag with `config.yml` as the default value

**Review by cevian — CHANGES_REQUESTED** [review]
Looks good. But, I'd like some test of the fact that this parses the config file(lowest priority), env, and cli flags (highest priority) in the right priority.

File: pkg/runner/runner.go (hunk 1 above)
Comment (antekresic) [reply pkg/runner/runner.go:86]: Yes, I simplified it for the draft, will add a flag for the filename.

**Review by cevian — APPROVED** [review]
LGTM

**Pull request merged**